Profile Section Documented (#5723)
chef/automate: components/docs-chef-io/content/automate/chef_automate_profile.md

+++

> title = "Profile"

Author: Profile is something which we can see in our Chef Automate, but yes totally agreed that 'User Profile' would be more findable

draft = false

gh_repo = "automate"
[menu]
  [menu.automate]
    title = "Profile"
    parent = "automate/settings"
    identifier = "automate/settings/chef_automate_profile.md Profile"
    weight = 120
+++

## Overview

The Chef Automate **Profile** help you set your personal information about the entities. You can set the _Display Name_, _Time Format_ and reset your _Password_.

> {{< figure src="/images/automate/chef-automate-profile.png" alt="Profile Section in Chef Automate">}}

Author: We do not have a specific name for the dropdown but yes we can add the full screen and outline this section

Select **Profile** to view the details. The **Details** tab lets you update your _Display Name_ and the _Time Format_, whereas the **Reset Password** tab lets you edit your password.

### Change a Display Name

> Change your display name by entering your preferred name. The display name gets updated as soon as you change the name in the bar and click **Save**.

Reviewer: ```suggestion
Change your display name by entering your preferred name. Select **Save** to update your name.
```

Reviewer: I'm concerned that my phrase `across Chef Automate.` is too colloquial. Does it make sense to you?

{{< figure src="/images/automate/display-name-in-profile.png" alt="Display Name of Chef Automate">}}

The Display Name bar should not be kept empty or else it throws error.

> {{< figure src="/images/automate/error-in-display-name-bar.png" alt="Error in Display Name">}}

Author: It gets updated at the same moment.

> {{< figure src="/images/automate/error-in-display-name-bar.png" alt="Error in Display Name">}}

Reviewer: That's important. Please list all of the things that get updated.

> ### Update the Date/Time Format

Reviewer: ```suggestion
### Change the Date/Time Format
```

> You can select the date and time format amongst the five formats provided in Chef Automate.

Reviewer: ```suggestion
You can select one of five date and time formats.
```

Are we changing the time zone or just the formatting? It isn't clear to me what the user can do.

Reviewer: Also, it doesn't look like you removed the old section from the docs.

Author: We can just change the formatting

> You can select one of five date and time formats from the drop-down. The date/time format will get updated as soon as you choose anyone and click **Save**.

Reviewer: ```suggestion
Select one of the five date and time formats from the drop-down menu. Select **Save** to change the appearance of the date and time across Chef Automate.
```

{{< figure src="/images/automate/date-time-format-in-profile.png" alt="Date/Time Format">}}

> Select the date/time format and save to get the changes reflected in the interface.

Author: It gets overwrite. We will be able to see the changed time format

> Select the date/time format and save to get the changes reflected in the interface.

Reviewer: That's important--and something that the user should know!

+++
title = "User Profile"

draft = false

gh_repo = "automate"
[menu]
  [menu.automate]
    title = "User Profile"
    parent = "automate/settings"
    identifier = "automate/settings/chef_automate_profile.md User Profile"
    weight = 120
+++

## Overview

Chef Automate users can manage their own display name, date/time format, and password. Select the user icon in the top navigation bar, and then select **Profile** from the drop-down menu.

{{< figure src="/images/automate/chef-automate-profile.png" alt="Profile Section in Chef Automate">}}

> Select **Profile** to view the details with respect to your _username_. The **Details** tab lets you update your _Display Name_ and the _Time Format_, whereas the **Reset Password** tab lets you update your password.

Reviewer: ```suggestion
Select **Profile** to view your _username_. The **Details** tab lets you update your _Display Name_ and the _Time Format_, whereas the **Reset Password** tab lets you update your password.
```
```suggestion
Select **Profile** to view the details with respect to your _username_. The **Details** tab lets you update your _Display Name_ and the _Time Format_, whereas the **Reset Password** tab lets you update your password.
```

Reviewer: Could you disambiguate between the `username` and the `Display Name`.  I don't know the difference between the two from this doc.

Author: Yes!
`username` is the login name which we need to login into our account, whereas display name is the name displayed in profiles.

### Change a Display Name

> You can change the display name for your Chef Automate in the bar.

Reviewer: ```suggestion
Change your display name by entering your preferred name.
```

{{< figure src="/images/automate/display-name-in-profile.png" alt="Display Name of Chef Automate">}}

{{< note >}}

> The **Display Name** gets updated as soon as you change the name in the bar.

Reviewer: Don't use a note for this. Put the content above the image. 

Also, I would think that the display name changes when the user hits `save`?

Author: Yes!
Will update both the pointers

{{< /note >}}

> The Display Name bar should not be kept empty or else it throws error.

Reviewer: ```suggestion
Every user needs a display name. An empty display name causes an error.
```
```suggestion
The Display Name bar should not be kept empty or else it throws error.
```

{{< figure src="/images/automate/error-in-display-name-bar.png" alt="Error in Display Name">}}

### Change the Date/Time Format

You can select one of five date and time formats.

{{< figure src="/images/automate/date-time-format-in-profile.png" alt="Date/Time Format">}}

Select the date/time format and save to get the changes reflected in the interface.

{{< note >}}

> The **Date/Time Format** gets updated as soon as you select a new time format from the drop down.

Reviewer: Again, wouldn't the user need to `save` first?

Author: Yes!
Have updated the same

{{< /note >}}

### Reset Password

> You can reset the password of Chef Automate login from the **Reset Password** tab in Profile.

Reviewer: ```suggestion
Reset your password of Chef Automate login from the **Reset Password** tab in Profile.
```
